backend/app.js

```diff
@@ -17,6 +17,8 @@ const collabRouter = require('./routers/collabRouter')
 const tokenController = require('./util/authCheck')
 const autoCompleteRouter = require('./routers/autoCompleteRouter')
 const homeRouter = require('./routers/homeRouter')
+const eventRouter = require('./routers/eventRouter')
+const notificationRouter = require('./routers/notificationRouter')

```

Comment:
notification router is NOT implemented in this branch, but the code is written here and it doesn't even run.

Reply:
Strange, it did work on me,
Removed the notification router and /notification endpoint from app.js

Reply:
Since you are working on multiple branches and it is implemented on the other branch, it might have worked fine, but if we merge this now, server would fail to deploy the code and website would be down.